command/gist.go

```diff
@@ -27,20 +27,29 @@ var gistCmd = &cobra.Command{
 }
 
 var gistCreateCmd = &cobra.Command{
-	Use:   `create {<filename>|-}...`,
```

Comment:
`-` cannot be repeated multiple times, therefore a more precise syntax would be `{<filename>...|-}`. Furthermore, the argument is optional in piped mode, so `[<filename>... | -]`